src/core/state/nonce/index.ts

```diff
@@ -54,6 +55,9 @@ export const nonceStore = createStore<CurrentNonceState>(
       const { nonces } = get();
       return nonces[address]?.[chainId] ?? null;
     },
+    clearNonces: () => {
+      set({ nonces: {} });
```

Comment:
do we want to clear *all* nonces or just the current wallet ones?

Reply:
All is fine IMO